nba-reference/eslint.config.mjs

```
@@ -35,67 +56,206 @@ const eslintConfig = defineConfig([
         {
-          allowExpressions: true,
-          allowTypedFunctionExpressions: true,
+          allowString: false,
+          allowNumber: false,
+          allowNullableObject: true,
         },
       ],
-      // No explicit any
-      '@typescript-eslint/no-explicit-any': 'error',
-      // Prefer nullish coalescing
       '@typescript-eslint/prefer-nullish-coalescing': 'error',
```

Biome (2.4.4)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)